l10n_ch_sepa/l10n_ch/template/pain.001.001.03.ch.02.xml.mako

```diff
@@ -58,15 +58,13 @@
    <%
    line=sepa_context['line']
    %>
-   % if line.bank_id.state == 'bvr':
-          <RmtInf>
-            <Strd>
-              <CdtrRefInf>
-                <Ref>${line.communication.replace(' ', '')}</Ref>
-              </CdtrRefInf>
-            </Strd>
-          </RmtInf>
-   % endif
+  <RmtInf>
+    <Strd>
```

Comment:
Strd is for structured references only. I don't think we should passe a free reference that way.